tasks/development.ts

```diff
@@ -208,4 +224,165 @@ export function load() {
     await run(ethsAddress, wrbtcAddress, "eth", "btc");
     await run(rusdtAddress, wrbtcAddress, "usd", "btc");
   });
+
+  task("dev:deploy-timelock")
+    .addParam("proposer", "proposer address")
+    .addParam("owner", "destinination owner address (usually multi-sig)")
+    .setAction(async ({ proposer, owner }, hre) => {
+      const timeLockFactory = await hre.ethers.getContractFactory("Timelock");
```

Comment:
If it's dev only, can we add an if condition if it's `mainnetFork` and if not throw an error?

Reply:
Nope, it's totally harmless and It's actually useful to be able to use it on RSK (e.g. I did tests on RSK deployed contracts)